README.md

```diff
@@ -143,7 +145,7 @@ These Firefox extensions can help prevent connections to Google domains and also
   - [JottaCloud](https://jottacloud.com/en/) - **9-eyes** - Norway based cloud storage provider with free and paid plans. Thanks @mynamesleon
 - Calendar
   - [Lightning Calendar](https://www.thunderbird.net/en-US/calendar/) (Thunderbird)- Developed by Mozilla Foundation and open-source. Integrates into Thunderbird and SeaMonkey.
-  - [Tutanota Calendar](https://tutanota.com/calendar/) - **14-eyes** - Developed by Tutanota. Free cross-platform encrypted calendar. Allows importing of other Calendar files.
+  - [Tutanota Calendar](https://tutanota.com/calendar/) - **14-eyes** - Developed by Tutanota. Free cross-platform encrypted calendar. Allows importing of other Calendar files. Can't be synced to other calendar software.
```

Comment:
This is a good point that comes up on the [r/tutanota subreddit](https://old.reddit.com/r/tutanota) every few weeks. Definitely worth mentioning 👍